values.yaml

```diff
@@ -231,33 +234,92 @@ zabbixagent:
-  #DB_SERVER_PORT: 5432
-  # -- User of database
-  #POSTGRES_USER: zabbix
-  # -- Password of database
-  #POSTGRES_PASSWORD: zabbix
-  # -- Name of database
-  #POSTGRES_DB: zabbix
+  # -- Certificate containing certificates for SAML configuration
+  #saml_certs_secret_name: zabbix-web-samlcerts
   service:
     # -- Type of service for Zabbix web
-    type: NodePort
+    type: ClusterIP
     # -- Cluster IP for Zabbix web
     clusterIP:
     # -- Port to expose service
     port: 80
+    # -- NodePort port to allocate (only if service.type = NodePort)
+    #nodePort: 31080
+    # -- Annotations for the zabbix-web service
+    annotations: {}
+    # metallb.universe.tf/address-pool: production-public-ips
+  # -- Extra environment variables. A list of additional environment variables. See example: https://github.com/cetic/helm-zabbix/blob/master/docs/example/kind/values.yaml
+  extraEnv: []
+    #- name: ZBX_SSO_SETTINGS
+    #  value: '{"baseurl": "https://zabbix.example.com"}'
+    #- name: ZBX_SERVER_NAME
+    #  value: Demo Zabbix
+  livenessProbe:
+    # -- Path of health check of application
+    path: /
+    # -- Number of seconds after the container has started before liveness
+    initialDelaySeconds: 30
+    # -- Specifies that the kubelet should perform a liveness probe every N seconds
+    periodSeconds: 10
+    # -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # -- When a probe fails, Kubernetes will try failureThreshold times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready
+    failureThreshold: 6
+    # -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  readinessProbe:
+    # -- Path of health check of application
+    path: /
+    # -- Number of seconds after the container has started before readiness
+    initialDelaySeconds: 5
+    # -- Specifies that the kubelet should perform a readiness probe every N seconds
+    periodSeconds: 10
+    # -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # -- When a probe fails, Kubernetes will try failureThreshold times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready
+    failureThreshold: 6
+    # -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+
+# **Zabbix Web Service** configurations
+zabbixwebservice:
+  # -- Enables use of **Zabbix Web Service**
+  enabled: true
+  replicaCount: 1
+  # -- set permissive podAntiAffinity to spread replicas over cluster nodes if replicaCount>1
+  pod_anti_affinity: true
+  resources: {}
+  image:
+    # -- Zabbix web Docker image name
+    repository: zabbix/zabbix-web-service
+    # -- Tag of Docker image of Zabbix web
+    #tag: ubuntu-6.0.5
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line can be removed, correct?

Reply:
commit 3f529c7